extras/create_snapshots_reports_scorecard.py

```diff
@@ -890,6 +913,9 @@ def main():
                 "No previous CybEx Scorecard JSON file found - continuing without creating CybEx Scorecard"
             )
 
+        global reports_to_generate
```

Comment:
Loop iterations question.

Reply:
This one is not in a loop, so I think it's a false alarm.

Reply:
Ah you are correct. My mistake!